src/store/modules/user/actions.ts

```diff
@@ -52,14 +52,30 @@ const actions: ActionTree<UserState, RootState> = {
   /**
    * Get User profile
    */
-  async getProfile ( { commit }) {
+  async getProfile ( { commit, dispatch }) {
     const resp = await UserService.getProfile()
     if (resp.status === 200) {
       const localTimeZone = moment.tz.guess();
       if (resp.data.userTimeZone !== localTimeZone) {
         emitter.emit('timeZoneDifferent', { profileTimeZone: resp.data.userTimeZone, localTimeZone});
       }
+      if(resp.data.facilities.length > 0) {
+        const payload = {
+          "inputFields": {
+            "facilityId": resp.data.facilities[0].facilityId
+          },
+          "fieldList": [],
```

Comment:
I think we should add fields if we just want to use some specific fields.

Reply:
Added all fields to fetch from API in fieldList.